site/public/css/polls.css

```diff
@@ -116,6 +116,23 @@ form {
     margin-left: 0.5em;
 }
 
+#poll-timer {
+    flex-grow: 100;
+    font-size: 14px;
+    margin-bottom: 10px;
+}
+
+#poll-hours{
+    width: 60px;
+    margin-right: 8px;
+    margin-bottom: 10px;
+}
+
+#poll-minutes{
+    width: 60px;
+    margin-bottom: 10px;    
+}
```

Comment:
It is generally better to use text-relative units (`em`) instead of raw pixel values for things like this.  I believe you can do this with the existing CSS code for the page though, so this shouldn't be needed anyway.

Reply:
I removed the poll-hours and poll-minutes and concise it to just poll-timer. And I switched from pixels to em. 
I didn't find anything else in the poll.css with small boxes I could put into.